include/pmacc/mappings/threads/ForEachIdx.hpp

```diff
@@ -75,7 +77,7 @@ namespace threads
          * The functor must fulfill the following interface:
          * @code
          * template< typename ... T_Args >
-         * void operator()( uint32_t const linearIdx, uint32_t const idx, T_Args && ... );
+         * void operator()( uint32_t linearIdx, uint32_t idx, T_Args && ... );
```

Comment:
Note: these removed `const`s are *not* part of the interface, which this comment is about, but implementation.

Reply:
But it's kind of an example people can clone from, so let's suggest `const` to them, no?

Reply:
Ah, I did not think of that and approached this more formally (then adding `const` here does not change the function signature and so `const` is not part of the interface). Will revert.